cypress/e2e/BiographyTestsFrontEnd/index.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('page title is {string}',(title)=>{
+cy.get(`h1:contains("${title}")`).should('be.visible').and('include.text',title)
+})
+
+And('the individuals title is {string}',(chiefCommunicationTeam)=>{
+cy.get(`div:contains("${chiefCommunicationTeam}")`).should('be.visible').and('include.text',chiefCommunicationTeam)
+})
+
+And('the following organizations are listed for this individual',()=>{
+cy.get('.profile-org').should('be.visible')
```

Comment:
1. Why is the code duplicated?
2. Where is a dataTable? Scenario step passes organization title via data table, you are missing it here, e.g. you are not verifying what the step is supposed to verify